kroxylicious-filters/kroxylicious-authorization/src/main/java/io/kroxylicious/filter/authorization/DescribeConfigsEnforcement.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.filter.authorization;
+
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletionStage;
+
+import org.apache.kafka.common.message.DescribeClusterResponseData;
+import org.apache.kafka.common.message.DescribeConfigsRequestData;
+import org.apache.kafka.common.message.DescribeConfigsResponseData;
+import org.apache.kafka.common.message.DescribeConfigsResponseData.DescribeConfigsResult;
+import org.apache.kafka.common.message.RequestHeaderData;
+import org.apache.kafka.common.protocol.Errors;
+import org.apache.kafka.common.resource.ResourceType;
+
+import io.kroxylicious.authorizer.service.Action;
+import io.kroxylicious.authorizer.service.Decision;
+import io.kroxylicious.proxy.filter.FilterContext;
+import io.kroxylicious.proxy.filter.RequestFilterResult;
+
+import static org.apache.kafka.common.resource.ResourceType.TOPIC;
+
+class DescribeConfigsEnforcement extends ApiEnforcement<DescribeConfigsRequestData, DescribeClusterResponseData> {
+    @Override
+    short minSupportedVersion() {
+        return 1;
+    }
+
+    @Override
+    short maxSupportedVersion() {
+        return 4;
+    }
+
+    @Override
+    CompletionStage<RequestFilterResult> onRequest(RequestHeaderData header, DescribeConfigsRequestData request, FilterContext context,
+                                                   AuthorizationFilter authorizationFilter) {
+        List<DescribeConfigsRequestData.DescribeConfigsResource> topicRequests = request.resources().stream()
+                .filter(resource -> ResourceType.fromCode(resource.resourceType()) == TOPIC).toList();
```

Comment:
We are not considering CLUSTER at the moment, right?

Reply:
Correct.